pkgs/tools/misc/bloaty/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, binutils, fetchgit }:
+
+stdenv.mkDerivation rec {
+  version = "2016.11.07";
+  name = "bloaty-${version}";
+
+  src = fetchgit {
```

Comment:
`fetchFromGitHub` won't work, as bloaty needs fetch submodules. Would be good to add `fetchSubmodules = true;` to `fetchgit` arguments. (Be explicit, so nobody asks why not `fetchFromGitHub`. :slightly_smiling_face:)